### DISCUSSION TITLE
License: Free for academia

### DISCUSSION THREAD

**Discussion by antonkesy** [BehaviorTree/Groot2#38] [Q&A]
I'm a bit confused about the license for academic purposes.

https://www.behaviortree.dev/groot/ states that:
> Free for academia: if you are a student or researcher, you can receive a complementary 1 year license.[ Fill this form](https://airtable.com/appqTlC23AiSoB4Fd/shrsC33d8feHZmKXs) and tell us more about your project.

How can I understand that:
- Free but only for 1 year (even for academic purposes)? 
- Always free for academics + 1 year free for outside usage?

Some clarification would be nice :) Thank you ❤️ 

**Background:**
We would like to use BT.CPPv4+ and Groot2 for ongoing university projects.
They've been around for more than a decade with many long-time contributors.
Buying _PRO (floating license)_ for each of them is not an option.
We'll have to stick to BT.CPPv3 :/

_Btw
Shouldn't it say `complimentary` instead of `complementary`_

**Comment by zhangkaiyuan007** [comment]
I was wondering if this state is still supported or if it has been discontinued. Did you solve your problem?

**Reply by antonkesy** [reply]
Sadly, nothing has changed for us. We still use BTv3.8 because of the _new_ pro licencing model. Additionally, we use an internal upgraded fork of Groot ...

**Reply by zhangkaiyuan007** [reply]
I see. Thanks for the answer.
